src/core/builder/solc.ts

```diff
@@ -110,7 +110,7 @@ export class SolcBuilder extends BaseBuilder {
       const contractInfo = JSON.parse(contractJsonFile);
       const bytecode = contractInfo.evm.bytecode.object;
       const deployedBytecode = contractInfo.evm.deployedBytecode.object;
-      const metadata = contractInfo.metadata;
+      const metadata = JSON.parse(contractInfo.metadata);
```

Comment:
Good catch, but I actually need to pass the raw data at the end of this function (iirc).

Safest i think would be to have both parsed and unparsed metadata here. I can double check tomorrow

Reply:
changed it now

Reply:
Don't see any changes?